src/main/scala/chisel3/Driver.scala

```diff
@@ -95,6 +95,18 @@ object Driver extends BackendCompilationUtilities {
 
   def emit[T <: RawModule](ir: Circuit): String = Emitter.emit(ir)
 
+  /** Elaborates the Module specified in the gen function into Verilog
+    *
+    *  @param gen a function that creates a Module hierarchy
+    *  @return the resulting String containing the design in Verilog
+    */
+  def emitVerilog[T <: RawModule](gen: => T): String = {
+    execute(Array[String](), { () => gen }) match {
```

Comment:
Do we ever care about wanting to pass arguments to emitVerilog? (e.g. optional argument)

Reply:
Seems like an options manager or an array of string should be passed in.  overloaded definitions for each one